chainer/training/extensions/variable_statistics_plot.py

```diff
@@ -163,9 +166,10 @@ class VariableStatisticsPlot(extension.Extension):
 
     def __init__(self, targets, max_sample_size=1000,
                  report_data=True, report_grad=True,
-                 trigger=(1, 'epoch'), file_name='statistics.png',
-                 percentile=(
+                 plot_mean=True, plot_std=True, plot_percentile=True,
```

Comment:
Is `plot_percentile` required? Is `percentile_sigmas=None` enough?

Reply:
It was added for consistency. I thought this would make the API easier to understand but maybe it's redundant. I'll fix.

Reply:
In fact, you could allow the user to set `plot_percentile=True` while setting/having by default `percentile_sigmas=None`, which would then default to `(0, 0.13, 2.28, 15.87, 50, 84.13, 97.72, 99.87, 100)` inside the `__init__` body. But again, unnecessary complicated?